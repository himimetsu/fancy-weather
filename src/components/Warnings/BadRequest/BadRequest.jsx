import React from 'react';
import './BadRequest.css';

const BadRequest = ({ lang }) => {
  return (
    <div className='bad-request'>
      {lang === 'ru'
        ? ('Пожалуйста, введите корректный запрос!')
          : lang === 'en'
            ? ('Please, enter the correct query!')
            : ('Будь ласка, введіть коректний запит!')
      }
    </div>
  );
};

export default BadRequest;
