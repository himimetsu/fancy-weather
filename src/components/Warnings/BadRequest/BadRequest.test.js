import React from 'react';
import ReactDOM from 'react-dom';
import BadRequest from './BadRequest';
import { render, cleanup } from '@testing-library/react';

afterEach(cleanup);

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<BadRequest />, div);
});

it('mathes snapshot', () => {
  const { asFragment } = render(<BadRequest />);
  expect(asFragment()).toMatchSnapshot();
})