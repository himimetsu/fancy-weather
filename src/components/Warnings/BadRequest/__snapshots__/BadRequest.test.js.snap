// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mathes snapshot 1`] = `
<DocumentFragment>
  <div
    class="bad-request"
  >
    <span
      class="ru"
    >
      Пожалуйста, введите корректный запрос!
    </span>
    <span
      class="en"
    >
      Please, enter the correct query!
    </span>
    <span
      class="be"
    >
      Калі ласка, увядзіце правільны запыт!
    </span>
  </div>
</DocumentFragment>
`;
